UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -312,6 +314,7 @@ public void WhenFssResponseNotFoundForScsProducts_ThenReturnFulfilmentException(
             HttpMethod httpMethodParam = null;
             string correlationIdParam = null;
             var searchBatchResponse = GetSearchBatchResponse();
+            CancellationTokenSource cts = new CancellationTokenSource();
```

Comment:
Instead of declaring it individually in test cases can we declare it as public variable and use the same variable.

public CancellationTokenSource cts = new CancellationTokenSource();